countdown.js

```diff
@@ -0,0 +1,5 @@
+//This loop starts from number 10 and decrement by 1 until it prints 0
+for(let i=10; i>=0; i--)
+  {
+  console.log(i);
+  }
```

Comment:
the opening bracket belongs on the same line as the for loop, as well you need spaces after the for word, and the assignment/comparison operators.

```
for (let i = 10; i >= 0; i--) {
  console.log(i);
}
```